spec/factories/budgets.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
```

Comment:
* Kindly fix this error i get when i try to run test 
![image](https://user-images.githubusercontent.com/52497246/144712598-da8f70cb-9068-438b-95f7-6586e207990e.png)
![Uploading image.png…]()

Reply:
fixed it

Reply:
![apps](https://user-images.githubusercontent.com/54015740/144716658-7f3fa2e8-c439-4c38-bccb-aa86cc910764.png)